packages/react/src/VideoPlayer/VideoPlayer.features.stories.tsx

```diff
@@ -3,6 +3,10 @@ import {Meta} from '@storybook/react'
 import posterImage from '../fixtures/images/example-poster.png'
 
 import {VideoPlayer} from '.'
+import {Stack} from '../Stack'
+import {Button} from '../Button'
+import {useVideo} from './hooks'
+import {PlayIcon} from '@primer/octicons-react'
```

Comment:
Nit.. could you put this at the top of the import order alongside other 3rd party deps instead of our local imports please

Reply:
Well spotted, thanks.